src/coffee/commands/images-convert.coffee

```diff
@@ -12,6 +12,7 @@ program
 .option '-d, --descriptions <path>', 'set image descriptions file path'
 .option '-r, --regex [name]', 'an optional RegExp used for filtering listed files (e.g.: /(.*)\.jpg/)', ''
 .option '-l, --logFile <path>', 'optionally log to a file instead of printing to console (errors will still be printed to stderr)'
+.option '-m, --minify', 'optionally minify (lossless compress) images', false
```

Comment:
why is it called minify and not compress? Minify were always used in context of resources like CSS or JS